lib/handlers/router-entities/route-caching/cached-routes-configuration.ts

```diff
@@ -147,4 +147,28 @@ export const CACHED_ROUTES_CONFIGURATION: Map<string, CachedRoutesStrategy> = ne
       ],
     }),
   ],
+  [
+    new PairTradeTypeChainId({
+      tokenIn: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
+      tokenOut: '*', // ANY TOKEN
+      tradeType: TradeType.EXACT_INPUT,
+      chainId: ChainId.MAINNET,
+    }).toString(),
+    new CachedRoutesStrategy({
+      pair: 'USDC/*',
+      tradeType: TradeType.EXACT_INPUT,
+      chainId: ChainId.MAINNET,
+      buckets: [
+        new CachedRoutesBucket({ bucket: 500, cacheMode: CacheMode.Tapcompare }),
```

Comment:
could you also add the `withLastNCachedRoutes` option to these? I think 10 might be more than enough

Reply:
oops ty, added